tasks/migrate-certificates.ts

```diff
@@ -33,7 +33,7 @@ interface MigratedCertificate {
   };
   ids: string[];
   amounts: number[];
-  txReceipt?: TransactionReceipt;
+  txReceipt?: { transactionHash?: string; status?: number };
```

Comment:
when including the full transaction receipt for every single certificate, the resulting output object was too big to stringify! So here I'm including the only two pieces of the transaction receipt that are used by the datastore update.  Since you can always retrieve a receipt by its hash if we need more details in the future, the hash should suffice.